File: problems/largest-element/boilerplate-full/function.cpp
#include <iostream>
#include <vector>

int main() {
    int n;
    std::cin >> n;

    std::vector<int> nums(n);
    for (int& number : nums) {
        std::cin >> number;
    }

    int largest = nums[0];
    for (int number : nums) {
        if (number > largest) {
            largest = number;
        }
    }

    std::cout << largest << '\n';
    return 0;
}

File: problems/largest-element/boilerplate-full/function.js
const fs = require('fs');

const values = fs.readFileSync(0, 'utf8').trim().split(/\s+/).map(Number);
const n = values[0];
const nums = values.slice(1, n + 1);

let largest = nums[0];
for (const number of nums.slice(1)) {
  if (number > largest) {
    largest = number;
  }
}

console.log(largest);
